linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml

```diff
@@ -47,6 +47,9 @@ ocil: |-
 
 fixtext: |-
     Configure Libreswan to use the system cryptographic policy.
+
+    If the ipsec service is not installed is not applicable.
```

Comment:
```suggestion
    If the IPSec service is not installed, this check is Not Applicable.
```

Reply:
or maybe this piece of information should be in OCIL clause instead. In RHEL8 STIG this usually goes the check content part.